source/blog/newpost.md

```diff
@@ -0,0 +1 @@
+This is a test blog post. Getting to learn GitHub the real way. 
```

Comment:
@ rollandf. Correct, this ("This is a test post. Getting to learn..) is an unrelated file. I am new to Git and GitHub, hence I am making some errors along the way.